aggregation-worker/src/main/java/au/org/emii/aggregator/au/org/emii/aggregator/config/AggregationOverridesReader.java

```diff
@@ -27,15 +29,18 @@
 
     private static Logger logger = LoggerFactory.getLogger(AggregationOverridesReader.class);
 
-    public static AggregationOverrides getAggregationOverrides(String s3Bucket, String s3Key, String layer) {
+    public static AggregationOverrides getAggregationOverrides(String url, String layer) {
         AggregationOverrides overrides = new AggregationOverrides();
 
-        logger.info("Loading aggregation templates file from S3.  Bucket [" + s3Bucket + "], Key [" + s3Key + "]");
+        logger.info("Loading aggregation templates file from url: " + url);
 
         //  Read config file from S3
         String overrideTemplateDocument = null;
         try {
-            overrideTemplateDocument = S3Utils.readS3ObjectAsString(s3Bucket, s3Key);
+            InputStream inputStream = new URL(url).openStream();
+
+            overrideTemplateDocument =  IOUtils.toString(inputStream);
+            inputStream.close();
```

Comment:
Better to close this in a try resources block (just add the openStream to the try above)